projects/15_basic_material/main.cpp

```diff
@@ -132,7 +135,11 @@ class ProjApp
     grfx::SamplerPtr                  mSampler;
     MaterialResources                 mWoodMaterial;
     MaterialResources                 mTilesMaterial;
-    MaterialResources                 mAltimeterMaterial;
+    MaterialResources                 mStoneWallMaterial;
+    MaterialResources                 mMeasuringTapeMaterial;
```

Comment:
Same for the materials.

Reply:
These aren't strictly needed but MaterialResources hangs on to some TexturePtr objects which would get auto destroy when device exists anyway...but if we don't keep them, it looks like we're purposely dangling pointers. I kept them for now but maybe a future PR can help fix that.